src/templates/rpc/warnet-rpc-statefulset.yaml

```diff
@@ -17,7 +17,7 @@ spec:
       containers:
       - name: warnet-rpc
         imagePullPolicy: Always
-        image: bitcoindevproject/warnet-rpc:0.2
+        image: bitcoindevproject/warnet-rpc:latest
```

Comment:
Just a note to say that this means if we push semantic versioned images we will also need to push an image with the tag `latest` explicitly for this to work.

Reply:
and I see you have the option for that in the build script.